ArkhamOverlay/Utils/IHasImageButton.cs

```diff
@@ -27,19 +27,26 @@ public static class HasImageButtonExtensions {
             }
         }
 
-        private static void DoLoadImage(IHasImageButton hasImageButton, string url) {
+        private static void DoLoadImage(IHasImageButton hasImageButton, string uri) {
             if (CardImageCache.ContainsKey(hasImageButton.Name)) {
                 hasImageButton.Image = CardImageCache[hasImageButton.Name];
                 CropImage(hasImageButton);
                 return;
             }
 
-            var bitmapImage = new BitmapImage(new Uri(url, UriKind.Absolute));
-            bitmapImage.DownloadCompleted += (s, e) => {
-                CardImageCache[hasImageButton.Name] = bitmapImage;
+            if (uri.StartsWith("http")) {
```

Comment:
You can create a Uri from the string and call `Uri.IsFile` since you'll create a Uri from it later anyways.

Reply:
Done